helpers/helper-translation-discussion.php

```diff
@@ -549,7 +549,8 @@ function( $comment_fields ) {
 
 		remove_action( 'comment_form_top', 'rosetta_comment_form_support_hint' );
 
-		$post = self::maybe_get_temporary_post( self::get_shadow_post_id( $this->data['original_id'] ) );
+		$post          = self::maybe_get_temporary_post( self::get_shadow_post_id( $this->data['original_id'] ) );
+		$mentions_list = apply_filters( 'wporg_load_mentions_list', array(), $comments, $this->data['locale_slug'], $this->data['original_id'] );
```

Comment:
```suggestion
		$mentions_list = apply_filters( 'gp_mentions_list', array(), $comments, $this->data['locale_slug'], $this->data['original_id'] );
```
I'd make it a `gp_` filter since this is not .org specific. We're just happening to use it from .org.